### DISCUSSION TITLE
Can KeePassXC-Browser extension used with KeePass 2 (or any other apps besides KeePassXC)?

### DISCUSSION THREAD

**Discussion by comiluv** [keepassxreboot/keepassxc#11035] [Q&A] [answered]
I was wondering if I could have a setup that utilizes KeePassXC-Browser extension installed in my web browser, and use KeePass 2 as my desktop password manager, so I could use KeePass 2 plugins with the browser extension.
Has anyone tried this combination or know if it works?
Thanks.

**Comment by droidmonkey** [comment, marked as answer]
Yes keepass2 has a native messaging plug-in that works with keepassxc
